### DISCUSSION TITLE
jwt.StandardClaims

### DISCUSSION THREAD

**Discussion by raliqala** [golang-jwt/jwt#150] [Q&A] [answered]
Hello  
Good day  

can I still use the jwt.StandardClaims ? or this is gone,

the registeredClaims is so dificult to use with gorm postgress

Its just a question I had.

Thanks in advance  

great package

**Comment by oxisto** [comment, marked as answer]
We do not have a fixed time when we will remove it, but that is the plan. The adjustments were needed because of several issues from the community about the way the old struct was handling time properties.

On one hand, we cannot guarantee that our structs work will every external package, I do however recognise that gorm is widely used. However, you might want to overthink, whether you it is a good idea to store tokens this way directly in the database. I will have a look in your other ticket wrt to gorm to see if there is an easy solution.

**Reply by mfridman** [reply]
As a best practice one would convert the JWT from this package into a well-defined object that makes sense in the target system. I'd strongly suggest decoupling the structs and avoid storing them as-is.

Fwiw, we will not do breaking changes to exported identifiers in the `/v4` version, and although we haven't discussed it yet I think we would continue to maintain the `/v4` for as long as possible in maintenance mode (bug fixes and security patches)?
